wolfssl/wolfcrypt/ecc.h

```diff
@@ -479,6 +479,9 @@ struct ecc_key {
 #ifdef WC_ECC_NONBLOCK
     ecc_nb_ctx_t* nb_ctx;
 #endif
+#ifdef HAVE_PKCS8
+    byte haspkcs8header;
```

Comment:
Put this in WOLFSSL_EVP_PKEY if possible.
Rename to hasPkcs8Header.

Reply:
Renamed. I cannot find way to put it into WOLFSSL_EVP_PKEY.